pkg/acceptance/compose_test.go

```diff
@@ -21,11 +21,13 @@ import (
 
 	"github.com/cockroachdb/cockroach/pkg/acceptance/cluster"
 	"github.com/cockroachdb/cockroach/pkg/build/bazel"
+	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 )
 
 const composeDir = "compose"
 
 func TestComposeGSS(t *testing.T) {
+	skip.WithIssue(t, 91420)
```

Comment:
This was later reverted via https://github.com/cockroachdb/cockroach/commit/c84d1a35432c9d4dcb4b3bcf17c0f65ac8fd590a

Compare via:
- https://github.com/cockroachdb/cockroach/commits/release-22.2.0/pkg/acceptance/compose_test.go
- https://github.com/cockroachdb/cockroach/commits/release-22.2/pkg/acceptance/compose_test.go